packages/components/src/components/PasswordCreationField/PasswordCreationField.tsx

```diff
@@ -0,0 +1,365 @@
+import type { ChangeEventHandler } from "react";
+import { useDeferredValue } from "react";
+import { useRef } from "react";
+import React, {
+  type PropsWithChildren,
+  startTransition,
+  useEffect,
+  useState,
+} from "react";
+import {
+  ClearPropsContext,
+  dynamic,
+  type PropsContext,
+  PropsContextProvider,
+} from "@/lib/propsContext";
+import {
+  flowComponent,
+  type FlowComponentProps,
+} from "@/lib/componentFactory/flowComponent";
+import styles from "./PasswordCreationField.module.scss";
+import * as Aria from "react-aria-components";
+import formFieldStyles from "@/components/FormField/FormField.module.scss";
+import clsx from "clsx";
+import { TunnelExit, TunnelProvider } from "@mittwald/react-tunnel";
+import { Policy } from "@mittwald/password-tools-js/policy";
+import { Generator } from "@mittwald/password-tools-js/generator";
+import Button from "@/components/Button";
+import { Action, type ActionFn } from "@/components/Action";
+import FieldLabel from "@/components/PasswordCreationField/components/FieldLabel/FieldLabel";
+import { IconHide, IconShow } from "@/components/Icon/components/icons";
+import {
+  getStatusFromPolicyValidationResult,
+  type ResolvedPolicyValidationResult,
+} from "@/components/PasswordCreationField/lib/getStatusFromPolicyValidationResult";
+import getStatusTextFromPolicyValidationResult from "@/components/PasswordCreationField/lib/getStatusTextFromPolicyValidationResult";
+import locales from "./locales/*.locale.json";
+import type { LocalizedStrings } from "react-aria";
+import { useLocalizedStringFormatter } from "react-aria";
+import generateValidationTranslation from "@/components/PasswordCreationField/lib/generateValidationTranslation";
+import { RuleType } from "@mittwald/password-tools-js/rules";
+import { FieldError } from "@/components/FieldError";
+import FieldDescription from "@/components/FieldDescription";
+import PromiseQueue from "p-queue";
+
+export const defaultPasswordCreationPolicy = Policy.fromDeclaration({
+  minComplexity: 3,
+  rules: [
+    {
+      ruleType: RuleType.length,
+      min: 12,
+    },
+    {
+      ruleType: RuleType.hibp,
+    },
+    {
+      identifier: "specialChars",
+      ruleType: RuleType.charPool,
+      charPools: ["special"],
+    },
+    {
+      identifier: "numberChars",
+      ruleType: RuleType.charPool,
+      charPools: ["numbers"],
+    },
+  ],
+});
+
+export interface PasswordCreationFieldProps
+  extends PropsWithChildren<
+      Omit<Aria.TextFieldProps, "children" | "value" | "defaultValue"> &
+        Partial<Pick<Aria.FieldErrorRenderProps, "validationErrors">>
+    >,
+    FlowComponentProps {
+  locales?: LocalizedStrings;
```

Comment:
Da bin ich mir nicht sicher, weil eigentlich wollen wir niemand zwingen unsere Art von Übersetzungen zu nehmen, deswegen haben wir sowas bisher noch nie in den Props gehabt. Aber da würd ich abwarten was Falki zu sagt, der wollte auch nochmal über alles drüber gucken